subprojects/common/math/src/main/kotlin/com/avito/math/Median.kt

```diff
@@ -0,0 +1,22 @@
+package com.avito.math
```

Comment:
I've seen an external library in previous commits? Why don't we use it?

Reply:
i removed external library and added inhouse implementation of median

Reply:
It was a small and tested library.
Why do we want an own implementation?

Reply:
you mean kotlin-statistics?
it is mostly a wrapper on top of commons-math, which is > 2mb